trigger/patch.go

```diff
@@ -194,6 +194,14 @@ func (t *patchTriggers) makeData(sub *event.Subscription) (*commonTemplateData,
 	if err != nil {
 		return nil, errors.Wrap(err, "getting collective status for patch")
 	}
+	grip.NoticeWhen(collectiveStatus != t.data.Status, message.Fields{
```

Comment:
> Add a log for when the patch's collective status differs from the event's status. If the bug described in [EVG-18561](https://jira.mongodb.org/browse/EVG-18561) is fixed, this should never log and the collective status query should be removed.

This was added as a fix to that bug, so I don't think it can be removed.

Reply:
I guess the issue with this fix is that it's not clear if the issue why the statuses differ? It seems like it's not supposed to change statuses once it's in a finished state, so fixing up the issue right before we send a notification seems too late.

Reply:
I agree. It was only added because the bug is not reproducible and not findable via code reading.